packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -119,6 +128,15 @@ export const EditTopicModal = ({
     });
   };
 
+  function removeImageMarkdown(desc: string): string {
```

Comment:
since we only use this function in one place, I think it is better to just move it into onDeleteImageClick like:

```typescript
const removeImageMarkdown = desc.replace(description, '').trim();
setDescription(removeImageMarkdown)
setImage(null)
```